package.json

```diff
@@ -14,6 +14,8 @@
     "cross-tab"
   ],
   "scripts": {
+    "dev": "vite serve ./demo --port 3000",
+    "start": "vite build ./demo && vite preview ./demo",
```

Comment:
Why we need `dev` and `start` if bfcache doesn’t work on `dev`?

Also, we can speed up `vite build` by disabling minification.

Reply:
I removed 'dev' script and disabled minification.